src/content/docs/shared/jest-missing-globals.mdx

```diff
@@ -24,15 +24,16 @@ Create a `jest.polyfills.js` file next to your `jest.config.js` with the followi
  * you don't want to deal with this.
  */
 
-const { TextDecoder, TextEncoder } = require('node:util')
+const { TextDecoder, TextEncoder, ReadableStream } = require("node:util")
```

Comment:
This is incorrect though. Must be imported from `node:stream/web`. That's the correct implementation of web's `ReadableStream`.